### PR TITLE
[14.0][MIG] sale_procurement_group_by_commitment_date

### PR DESCRIPTION
**Pull request by JuanyDForgeflow**
@ForgeFlow 

Migration to 14.0

This module creates different procurements groups for different commitment
dates in a sale order line when the sale order is confirmed.

### CODE REVIEW

**Comment by rousseldenis** [comment]
@JuanyDForgeflow Can you check travis ?

**Review by MiquelRForgeFlow — APPROVED** [review]

File: sale_procurement_group_by_commitment_date/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
```
Comment (MiquelRForgeFlow) [inline sale_procurement_group_by_commitment_date/__manifest__.py]:
```suggestion
# Copyright 2017 ForgeFlow S.L.
```

Comment (JuanyDForgeflow) [reply sale_procurement_group_by_commitment_date/__manifest__.py]: @MiquelRForgeFlow done

**Comment by JuanyDForgeflow** [comment]
@rousseldenis how do I check it? I see it green

**Comment by rousseldenis** [comment]
> @rousseldenis how do I check it? I see it green

We switched to Github actions, so Travis is no more a problem. :smiley:

**Comment by rousseldenis** [comment]
@JuanyDForgeflow Could you check conflicts?

**Comment by MiquelRForgeFlow** [comment]
@JuanyDForgeflow see like I have done in https://github.com/OCA/sale-workflow/pull/2084

**Comment by MiquelRForgeFlow** [comment]
conflicts

**Comment by MiquelRForgeFlow** [comment]
@rousseldenis could you review?

**Review by rousseldenis — APPROVED** [review]

**Comment by dreispt** [comment]
@JuanyDForgeflow Needs a rebase to resolve conflicts

**Comment by JuanyDForgeflow** [comment]
@dreispt I just rebased, the test errors are not related to this module, I think is ready now

**Review by AaronHForgeFlow — APPROVED** [review]
Functional review, LG!

**Comment by AaronHForgeFlow** [comment]
@dreispt I think this is fixed now :)

**Comment by AaronHForgeFlow** [comment]
@JuanyDForgeflow can you rebase?

**Comment by AaronHForgeFlow** [comment]
It seems the rebel modules config was not ok. Let's see.

**Comment by AaronHForgeFlow** [comment]
This is green now :) @rousseldenis  ok to merge?

**Pull request merged**